src/java.base/share/classes/sun/nio/ch/ThreadPool.java

```diff
@@ -70,7 +70,7 @@ int poolSize() {
 
     static ThreadFactory defaultThreadFactory() {
         return (Runnable r) -> {
-            Thread t = new Thread(r);
+            Thread t = InnocuousThread.newThread(r);
```

Comment:
This change is observable as the threads in this thread pool execute user code. I can't imagine anything really wanting to rely on the undocumented inheritance but it's possible so we'll need to create a CSR for this.

Reply:
Okay, I will create a CSR.

Reply:
[CSR](https://bugs.openjdk.org/browse/JDK-8346124) filed.

Reply:
I tweaked the CSR wording to make it clear that this is about inheritance.